tls/s2n_config.c

```diff
@@ -807,3 +802,17 @@ int s2n_config_set_cert_tiebreak_callback(struct s2n_config *config, s2n_cert_ti
     config->cert_tiebreak_cb = cert_tiebreak_cb;
     return 0;
 }
+
+int s2n_fetch_single_default_cert(struct s2n_config *config, struct s2n_cert_chain_and_key **cert_and_key)
```

Comment:
I think it's a bit unexpected to have this return the number of certificates. I suggest just spending a few bytes to keep a `num_certificates` count in `s2n_config` and then have this function use the typical 0, -1 return values.

Reply:
changed the function to return the pointer to the cert, also added another function to return the number of default certs. Counting the actual number of certs added to the store is bit tricky as we do not actually add all the certs due to the tie-breaking and/or multiple certs with the same domain name